tests/integration/high_availability/scripts/deploy_chaos_mesh.sh

```diff
@@ -11,7 +11,7 @@ if [ -z "${chaos_mesh_ns}" ]; then
 fi
 
 deploy_chaos_mesh() {
-	if sg microk8s -c 'microk8s.helm3 repo list' | grep -q 'chaos-mesh'; then
```

Comment:
I think adding the repo should be called without the test, because it does not error out when the repo is already added.